### PR TITLE
OOC is now logged in a discord channel

### PR DESCRIPTION
**Pull request by jupyterkat**
## About The Pull Request

Adds a discord channel for ooc, adds end lines to roundend messages

## Why It's Good For The Game

why not

## Changelog
:cl:
add: Added discord ooc channel
/:cl:

### CODE REVIEW

**Comment by TheFakeElon** [comment]
Why though

**Comment by Singularity69** [comment]
> Why though

why not

**Comment by TheFakeElon** [comment]
It makes ick ocking a much bigger problem.

**Comment by jupyterkat** [comment]
It’s your business to make the channel available to admins only or not anyways

**Comment by MCterra10** [comment]
requires updates to TGS config and discord

File: austation/code/modules/client/verbs/ooc.dm [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/client/ooc(msg as text)
+	. = ..()
+	if(holder?.fakekey)
+		send2chat("**[holder.fakekey]:** [msg2url(msg)]", "ooc")
+	else
+		send2chat("**[key]:** [msg2url(msg)]", "ooc")
```
Comment (MCterra10) [inline austation/code/modules/client/verbs/ooc.dm:6]: I wonder if I can make a byond account with the username @\here or @\everyone

**Review by MCterra10 — APPROVED** [review]

File: austation/code/modules/client/verbs/ooc.dm (hunk 1 above)
Comment (MCterra10) [reply austation/code/modules/client/verbs/ooc.dm:6]: image
most unfortunate

Comment (jupyterkat) [reply austation/code/modules/client/verbs/ooc.dm:6]: h a

**Review by TheFakeElon — APPROVED** [review]

**Pull request merged**